src/openeo_odc/map_processes_odc.py

```diff
@@ -103,7 +105,9 @@ def map_data(id, process, kwargs):
         if process_name != 'apply':
             params['reducer'] = {}
     else:
-        params['data'] = convert_from_node_parameter(params['data'],
+        params['data'] = convert_from_node_parameter(params['data'])
```

Comment:
Tests are currently failing, I think because you don't parse `kwargs['from_parameter']` as a second parameter to `convert_from_node_parameter`